src/constants/tokens.ts

```diff
@@ -136,9 +141,29 @@ export const IndexCoopBitcoin2xIndex: Token = {
   url: 'btc2x',
 }
 
-const eth2x = getTokenFromSymbol('ETH2X')!
+const btc3xArbitrum = getIndexTokenData('BTC3X', ARBITRUM.chainId)!
+export const IndexCoopBitcoin3xIndex: Token = {
+  ...btc3xArbitrum,
+  address: '',
```

Comment:
is this a TODO, or intend for the address to be an empty string?

Reply:
Intended to be empty - as it's not available on mainnet 👍